contracts/WildcardSteward_v2.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity 0.5.16;
+pragma solidity 0.5.15;
```

Comment:
why downgrading?

Reply:
We switched to buidler.dev for testing as its lightning fast. From what I've tried, it seemed like it only supported up to 0.5.15

Reply:
Yes, we could switch it to the latest version if we run the tests on ganache rather than buidler; and I guess for mainnet deploys we should probably do that (I mean why not use the latest versions?)